azure_devops_rust_api/examples/get_commits.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// get_commits.rs
+// Getting all the commits from a PR example.
+use anyhow::Result;
+use azure_devops_rust_api::git;
+use azure_devops_rust_api::Credential;
+use std::env;
+use std::sync::Arc;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    // Initialize logging
+    env_logger::init();
+
+    // Get authentication credential either from a PAT ("ADO_TOKEN") or via the az cli.
+    let credential = match env::var("ADO_TOKEN") {
+        Ok(token) => {
+            println!("INFO:Authenticate using PAT provided via $ADO_TOKEN");
+            Credential::from_pat(token)
+        }
+        Err(_) => {
+            println!("INFO:Authenticate using Azure CLI");
+            Credential::from_token_credential(Arc::new(azure_identity::AzureCliCredential {}))
+        }
+    };
+
+    // Get ADO server configuration via environment variables
+    let organization = env::var("ADO_ORGANIZATION").expect("Must define ADO_ORGANIZATION");
+    let repository_id = env::var("ADO_REPO_ID").expect("ADO_REPO_ID");
+    let pull_request_id: i32 = env::var("ADO_PR_ID").expect("ADO_PR_ID").parse().unwrap();
+    let project = env::var("ADO_PROJECT").expect("Must define ADO_PROJECT");
+
+    // Create a git client
+    let git_client = git::ClientBuilder::new(credential).build();
+
+    // Get Commits for the PR
+    let pr_commit_client = git_client
```

Comment:
naming: This variable is incorrectly named - it should be called `pr_commits` as it stores the result of the `get_pull_request_commits` operation.

Reply:
Updated in the latest [commit](https://github.com/microsoft/azure-devops-rust-api/pull/27/commits/276e698c731ca453693e3800bab1ecbdad2338dd)